src/content/blog/astro-4-beta.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: "Astro 4.0 Beta"
+description: "Astro 4.0 beta is released."
+publishDate: "November 22, 2023"
+authors:
+  - matthew
+coverImage: "/src/content/blog/_images/astro-4beta/post-header-4.0beta.webp"
+socialImage: "/src/content/blog/_images/astro-4beta/post-header-4.0beta.webp"
+lang: "en"
+---
+
+Today we've released the first beta of Astro 4! Those feeling adventurous can install the new beta, review our in-progress upgrade docs, and report any bugs you find to the [Astro discord](https://astro.build/chat).
+
+## What to expect in Astro 4
+
+We're releasing Astro 4 to stay up to date with the latest version of Vite, which just released Vite 5 last month.
+
+As Vite 5 is *also* a small major, most projects will be able to upgrade to Astro 4 with only minimal changes to your code, if any at all.
+
+Additionally, in Astro 4 we are planning to bring some of the new features we have been working on over the last few months out of experimental status and mark them as stable. We are still deciding *which* features those will be. Stay tuned for more!
```

Comment:
```suggestion
Additionally, in Astro 4 we are planning to bring some of the new features we have been working on over the last few months out of experimental status and mark them as stable. Stay tuned for more!
```

It's a funny line, but it also reads a bit like "ehh, we don't know what we're doing!" Would suggest removing